common/raw_hashtable_benchmark_helpers.h

```diff
@@ -151,6 +187,19 @@ struct CarbonHashDI<int> {
   }
 };
 
+template <int LowZeroBits>
+struct CarbonHashDI<LowZeroBitInt<LowZeroBits>> {
+  using IntT = LowZeroBitInt<LowZeroBits>;
+  static auto getEmptyKey() -> IntT { return -1; }
+  static auto getTombstoneKey() -> IntT { return -2; }
```

Comment:
Is it deliberate that these are the _shifted_ values -1 and -2, rather than the _literal_ values? (it's subtle to me with the implicit conversions)